resources/js/components/Board.js

```diff
@@ -420,6 +459,17 @@ export class Board extends React.Component {
                 onClick={this.handleBackspaceButtonClick}>
           Backspace
         </button>
+        
+
+        
+        <button className="swap-button" onClick={this.openSwapModal}>Swap Boards</button>
+        
+        
+        <button className="settings-button" onClick={this.backButtonFunc}>Settings</button>
+       
+
+
+
```

Comment:
probably don't need all this whitespace
```suggestion
        <button className="swap-button" onClick={this.openSwapModal}>Swap Boards</button>
        <button className="settings-button" onClick={this.backButtonFunc}>Settings</button>
```